lib/streamlit/elements/media_proto.py

```diff
@@ -0,0 +1,150 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018-2019 Streamlit Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Python 2/3 compatibility
+from __future__ import print_function, division, unicode_literals, absolute_import
+from future.types import newbytes
+from streamlit.compatibility import setup_2_3_shims
+
+setup_2_3_shims(globals())
+
+import io
+import base64
+import re
+
+from validators import url
+
+from streamlit.proto import Video_pb2
+
+
+def _reshape_youtube_url(url):
+    """Return whether URL is any kind of YouTube embed or watch link.  If so,
+    reshape URL into an embed link suitable for use in an iframe.
+
+    If not a YouTube URL, return None.
+
+    Parameters
+    ----------
+    url : str, bytes
```

Comment:
"str or bytes"
See https://numpydoc.readthedocs.io/en/latest/format.html

(Also, you don't have to write pydocs for private functions — unless the function name and arguments aren't particularly clear. Up to you, though!)